content/docs/start/experiments.md

```diff
@@ -54,92 +128,84 @@ reflected in the experiment run.
-$ dvc exp run --queue -S train.min_split=8 -S train.n_est=100
-Queued experiment 'c605382' for future execution.
-$ dvc exp run --queue -S train.min_split=64 -S train.n_est=100
-Queued experiment '0cdee86' for future execution.
+$ dvc exp run --queue -n cnn-48 -S model.cnn.conv_units=48
+$ dvc exp run --queue -n cnn-64 -S model.cnn.conv_units=64
+$ dvc exp run --queue -n cnn-96 -S model.cnn.conv_units=96
+$ dvc exp run --queue -n cnn-128 -S model.cnn.conv_units=128
 ```
 
 Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
 
 ```dvc
 $ dvc exp run --run-all --jobs 2
+...
+Reproduced experiment(s): cnn-128, cnn-96, cnn-64, cnn-48
+To apply the results of an experiment to your workspace run:
+
+	dvc exp apply <exp>
+
+To promote an experiment to a Git branch run:
+
+	dvc exp branch <exp>
 ```
 
+The command will run all queued experiments in a random order.
+
 ## Comparing many experiments
 
 To compare all of these experiments, we need more than `diff`. `dvc exp show`
 compares any number of experiments in one table:
 
 ```dvc
 $ dvc exp show --no-timestamp \
-               --include-params train.n_est,train.min_split
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
-│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
-│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
-│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
-│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
-│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
-│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
-│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
-│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
-└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+               --include-params model.cnn.conv_units \
+               --include-metrics categorical_accuracy
 ```
 
-Each experiment is given an arbitrary name by default (although we can specify
-one with `dvc exp run -n`.) We can see that `exp-98a96` performed best among
-both of our metrics, with 100 estimators and a minimum of 64 samples to split a
-node.
+![dvc exp show screenshot](/img/doc/start/exp-ss-36794.png)
+
+We can see that `cnn-128` performed best in `categorical_accuracy`.
 
 > See `dvc exp show --help` for more info on its options.
 
 ## Persisting experiments
 
-Now that we know the best parameters, let's keep that experiment and ignore the
-rest.
+The metrics to optimize are usually domain or problem dependent. In these
+experiments we are trying to optimize for `categorical_accuracy`. Let's keep
+experiment with the highest `categorical_accuracy` score and ignore the rest.
 
 `dvc exp apply` rolls back the <abbr>workspace<abbr> to the specified
```

Comment:
In retrospect, I'd prefer to introduce `dvc exp branch` here instead of `dvc exp apply`, but that's best left for a separate issue. cc @jorgeorpinel